recipes/filterpy/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "filterpy" %}
+{% set version = "0.1.5" %}
+{% set sha256 = "42d42d7aeef752e297839776ae01721e79bf446e3c17e3d48914c03b5fbdd816" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+  patches:
+    # the archive uploaded to pypi is missing the LICENSE file, this patch adds it back
+    - license.patch
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy x.x
+    - matplotlib
+    - scipy
+  run:
+    - python
+    - numpy x.x
+    - matplotlib
+    - scipy
+
+test:
+  imports:
+    - filterpy
+    - filterpy.common
+    - filterpy.discrete_bayes
+    - filterpy.examples
+    - filterpy.gh
+    - filterpy.hinfinity
+    - filterpy.kalman
+    - filterpy.leastsq
+    - filterpy.memory
+    - filterpy.monte_carlo
+    - filterpy.stats
+
+about:
+  home: https://github.com/rlabbe/filterpy
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Kalman filters and other optimal and non-optimal estimation filters in Python'
+
+  description: |
+      Kalman filtering and optimal estimation library in Python.
+      Kalman filter, Extended Kalman filter, Unscented Kalman filter,
+      g-h, least squares, H Infinity, smoothers, and more.
+      Has companion book 'Kalman and Bayesian Filters in Python'.
+  doc_url: https://filterpy.readthedocs.io/en/latest/
+  dev_url: https://github.com/rlabbe/filterpy
+
+extra:
+  recipe-maintainers:
+    - hajapy
+    - rlabbe
```

Comment:
We are incorporating the fixes in the patch upstream and so this PR can wait until those changes are in and the pypi package gets updated. As for the maintainer status of @rlabbe, I still am awaiting a verdict but if it ends up being the bottleneck on the PR later I'll do as you suggest.